jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/startupchecks/StartupCheckConfig.java

```diff
@@ -0,0 +1,17 @@
+package se.fortnox.reactivewizard.jaxrs.startupchecks;
+
+import se.fortnox.reactivewizard.config.Config;
+
+@Config("startupcheck")
+public class StartupCheckConfig {
+    private boolean failOnError = true;
```

Comment:
How come you need this to be configurable? Since it is bound to the code rather than to the environment - can you use an annotation or similar on the resource method to signal to the checker that it is ok with colliding paths? That would remove the dependency on the config module and would mean that you don't need to keep the configs up to date with the framework revision.

Reply:
Good idea. 

We only wanted the config so that we could ignore the error in the case there is some case we haven't thought about.
We plan to remove the config once we see there are no issues.